dev/com.ibm.ws.security.jwtsso/bnd.bnd

```diff
@@ -0,0 +1,132 @@
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: WebSphere Security JWT Builder and Consumer
+Bundle-SymbolicName: com.ibm.ws.security.jwtsso
+Bundle-Description: WebSphere Security JWT SSO Authentication, version=${bVersion}
+
+Web-ContextPath: /jwtsso
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+WS-TraceGroup: \
+  JWTSSO
+
+# For each exported package, create (in that package) a package-info.java...
+# file, and place an @version javadoc tag in package-level javadoc. 
+# Append ";provide:=true" if this bundle also provides an implementation
+# for the exported API.
+IBM-Default-Config: OSGI-INF/wlp/defaultInstances.xml
+
+Export-Package: \
+
+
+Import-Package: \
+    javax.security.auth, \
+    com.ibm.websphere.security.auth, \
+    com.ibm.websphere.security.cred, \
+    com.ibm.websphere.ras, \
+    com.ibm.websphere.ras.annotation, \
+    com.ibm.ejs.ras, \
+    com.ibm.websphere.crypto, \
+    com.ibm.ws.ras.instrument.annotation, \
+    com.ibm.websphere.logging.hpel, \
+    com.ibm.ws.ffdc, \
+    com.ibm.ws.security.common.random, \
+    com.ibm.ws.security.common.structures, \
+    com.ibm.ws.security.common.time, \
+    com.ibm.ws.security.common.token.propagation, \
+    com.ibm.ws.security.common.web, \
+    com.ibm.ws.security.common.jwk.impl, \
+    com.ibm.ws.security.common.jwk.interfaces, \
+    com.ibm.json.java.*, \
+    org.osgi.service.component.*, \
+    org.osgi.framework, \
+    org.osgi.util.tracker, \
+    com.ibm.wsspi.kernel.service.utils, \
+    com.ibm.websphere.kernel.server, \
+    org.osgi.framework.wiring, \
+    org.jose4j.*, \
+    org.apache.commons.codec.binary, \
+    javax.management, \
+    javax.servlet, \
+    javax.servlet.http, \
+    javax.net.ssl, \
+    org.apache.http.*;resolution:=optional, \
+    com.google.gson.*;resolution:=optional, \
+    javax.ws.rs.client;version="[2.0.0, 3.0.0)";resolution:=optional, \
+    javax.ws.rs.core;version="[2.0.0, 3.0.0)";resolution:=optional, \
+    com.ibm.wsspi.security.registry;version="[1.0.0, 2.0)";resolution:=optional, \
+    com.ibm.wsspi.ssl;version="[1.0.0, 2.0)";resolution:=optional, \
+    com.ibm.websphere.security;version="[1.0.0, 2.0)";resolution:=optional, \
+    com.ibm.websphere.ssl;version="[1.0.0, 2.0)";resolution:=optional, \
+    com.ibm.ws.ssl;version="[1.0.0, 2.0)";resolution:=optional, \
+    com.ibm.ws.security.wim;version="[1.0.0, 2.0)";resolution:=optional, \
+    com.ibm.wsspi.security.wim.model;version="[1.0.0, 2.0)";resolution:=optional
```

Comment:
We can clean up most of this stuff until it's actually needed in the code. (I actually thought this had already been cleaned up so all of the non-optional packages had been replaced with a single `*` line.) Same goes for the `buildpath` and the `testpath`.

Reply:
reduced it down to what we're likely to need.